.travis.yml

```diff
@@ -16,6 +16,7 @@ matrix:
     - env: JOB=HOUSEKEEPING
       python: 3.7
       install:
+        - pip install requests
         - ./bin/fetch-configlet
         - git clone https://github.com/exercism/problem-specifications spec
         - pip install -r requirements-generator.txt
```

Comment:
Maybe add requests to the requirements-generator.txt and bump it above the fetch-configlet call?

Reply:
That was my first idea, but as it isn't technically a requirement for the generator, I was hesitant to add it there. Likewise, creating yet another requirements file for a single package seemed wrong too.

Reply:
True ... not a big fan of having it there all alone, but I suppose it's acceptable.